examples/vpc-scenario-1/README.org

```diff
@@ -0,0 +1,40 @@
+** Example to test the VPC Scenario 1 Module
```

Comment:
It's a nit-pick, but just for consistency, it would be better to use `#` for headings, as that is what we're using everywhere else

Reply:
Should I be using markdown instead of Org?

Reply:
I can rewrite the README file as a markdown file instead of an Org file.